### ISSUE TITLE
Converting stream grids to vector layers produce honeycomb/lattice structures at DEM's level areas

### ISSUE THREAD

**Issue by tshindler** [dtarb/TauDEM#224]
All TauDEM tools that create stream grids from my filled DEMs leave multiple rows of cells with "1" in areas where slope is planar. When converting to a vector layer (shapefile) using ArcTool "Raster to Polyline", this results in thousands of little honeycomb or lattice shapes all over the layer, that take hours to clean up. Back when I had access to Spatial Analyst extension the "Stream to Feature" tool would not do that to a grid created using the ArcHydro tools. I can't find any way to get around this, and the issue does not seem to be reported by others. I see no tool in the TauDEM suite for converting a stream grid to a vector layer, so I assume it must be done with the "Raster to Polyline" tool. Am I missing something?

My grid is a lidar generated DEM with  9 foot cells. I've tried resampling to a coarser resolution, but the problem is only decreased, not eliminated. I use ArcGIS 10.2. (I have QGIS loaded, but am not very skilled in it.)

I've attached an example from a screen shot.

Thanks,

Tom 
![StreamLattice](https://user-images.githubusercontent.com/79135559/108038107-504ca800-6fef-11eb-83f2-f3b74e3740e8.jpg)

**Comment by dtarb** [comment]
Try the TauDEM Stream Reach and Watershed (Streamnet) function https://hydrology.usu.edu/taudem/taudem5/help53/StreamReachAndWatershed.html.

**Comment by tshindler** [comment]
David,
Your comment made me go back and look more carefully at that tool. I had used it, but didn't know it created a vector layer, because that layer doesn't get added to the map, like the grids do. Once I found that the order wasn't just a grid, but also a shapefile, that totally solved my problem, creating a clean stream layer. 

The arctool dialog for that tool adds a "dbf" to the filename for the vector layer, not "shp", so I thought it was just creating a table that I wasn't necessarily going to use. So, raising the profile of that output would be good, since that's really the main thing you want from that tool, and the only tool that creates the vector layer.

Thanks again for all your work on these tools, and for pointing me toward a closer look at that function.
Tom
